docker-compose.yml

```
@@ -0,0 +1,32 @@
+    ports:
+      - "8000:8000"
+    volumes:
+      - ./ai-service/app:/app/app
+    environment:
+      - PYTHONUNBUFFERED=1
+      - ENVIRONMENT=development
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost:8000/"]
+      interval: 30s
+      timeout: 10s
+      retries: 3 
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)